protocol/0014-ORDT-order_types.md

```diff
@@ -49,6 +49,117 @@ Notes on scope of current version of this spec:
+
+* `minimum peak size` - this determines when a glassberg order is eligible for refresh.
+The glassberg is refreshed any time the order's displayed quantity less than the minimum peak size.
+
+* `refresh policy` - this specifies when a glassberg order that is eligible for refresh and has remaining volume >0 is refreshed:
+
+    * `IMMEDIATE` - the refresh occurs _after_ processing the transaction that depleted the display quantity to zero.
+
+    * `BLOCK_END` - the rerresh occurs after processing the _entire block_ containing the transaction that depleted the display quantity to zero.
+
+
+#### Validity
+
+* The order's non-glassberg-related fields must be set so as to make a valid order.
+
+* Any persistent TIF (GTC, GTT, GFA, GFN) can be a glassberg order.
+
+* A glassberg order may have either an ordinary or pegged limit price. 
+Market glassberg orders are not supported, even if with a persistent TIF.
+
+* Glassbergs may be post only.
+
+* `initial peak size` must be greater than TODO
+
+* `minimum peak size` must be `>` 0 and `≤ initial peak size`
+
+
+#### Execution
+
+* Glassberg orders trade just like non-glassberg persistent order, as if the order entered the book with quantity = initial peak size on submission and again each time they are refreshed.
+That is:
+
+    * On entry, unlike normal orders, `displayed quantity` is set to `initial peak size` not `quantity`.
+    
+    * As for any other order, `remaning == quantity` on entry.
+
+* The maximum size for a trade involving a glassberg order is the `displayed quantity` immediataly prior to the trade.
+(This is technically also true for a normal order, given that for non-glassberg orders `displayed quantity == remaining`.)
+
+* When a glassberg order trades, both `remaining` and `displayed quantity` are reduced by the trade size.
+
+* Glassberg orders can trade many times without refresh, reducing `displayed quantity` each time.
+The order will not be refreshed after each trade while `displayed quantity ≥ minimum peak size`.
+
+* The order will also not be refreshed within a block even if multiple trades occur or `displayed quantity == 0` if the refresh policy is set to `BLOCK_END`. 
+
+* Glassberg orders never trade more than their `displayed quantity` at the start of the transaction, as the result of any one transaction.
+
+* When `displayed quantity < minimum peak size` and `remaining > displayed quantity` the order will be refreshed:
+
+    * The refresh either happens at the end of the transaction when the order became eligible for refresh, if the `refresh policy == IMMEDIATE`; or at the end of the block containing that transaction if the `refresh policy == BLOCK_END`
+
+    * On refresh `display quantity` is set to `min(remaining, initial peak size)`.
+
+* Once the remaning quantity is equal to the displayed quantity, no further refresh is possible.
+The order now behaves like a normal limit order and will leave the book if it trades away completely.
```

Comment:
I suppose on refresh all the other limit orders at that level have higher priority for execution (if we're doing the refresh at block end).

Reply:
Yes - latest commit this AM clarifies this and also that as increasing size amendments add to remaining but not display quantity they *do not lose time priority* unlike for normal orders.